pkgs/tools/networking/ntopng/default.nix

```diff
@@ -4,51 +4,75 @@
 , curl
 , expat
 , fetchFromGitHub
-, fetchpatch
+, git
 , json_c
 , libcap
 , libmaxminddb
 , libmysqlclient
 , libpcap
+, libsodium
 , ndpi
 , net-snmp
 , openssl
 , pkg-config
+, rdkafka
+, gtest
 , rrdtool
+, hiredis
 , sqlite
+, which
 , zeromq
 }:
 
 stdenv.mkDerivation rec {
   pname = "ntopng";
-  version = "5.2.1";
+  version = "5.6";
 
   src = fetchFromGitHub {
     owner = "ntop";
     repo = "ntopng";
-    rev = version;
-    sha256 = "sha256-FeRERSq8F3HEelUCkA6pgNNcP94xrWy6EbJgk+cEdqc=";
+    rev = "refs/tags/${version}";
+    hash = "sha256-E5/yrCaC406KlBBuy2Cue7PBXaN/kYfCZQyZSCgm9jA=";
+    leaveDotGit = true;
```

Comment:
AFAIK, leaveDotGit isn't reproducible, better not use it in nixpkgs.

Reply:
Thanks, remove as it was a left-over anyways.